graviti/dataframe/frame.py

```diff
@@ -283,11 +276,11 @@ def shape(self) -> Tuple[int, int]:
 
         Examples:
             >>> df = DataFrame(
-                    {
-                        "filename": ["a.jpg", "b.jpg", "c.jpg"],
-                        "box2ds": {"x": [1, 2, 3], "y": [1, 2, 3]},
-                    }
-                )
+            ...     {
+            ...         "filename": ["a.jpg", "b.jpg", "c.jpg"],
+            ...         "box2ds": {"x": [1, 2, 3], "y": [1, 2, 3]},
+            ...     }
+            ... )
```

Comment:
The construction in the example is different from the one in the `init` example? Should it be updated?